app/backend/src/clj/teet/asset/asset_commands.clj

```diff
@@ -21,11 +21,14 @@
    :project-id [:thk.project/id project-id]
    :authorization {:cost-items/edit-cost-items {}}
    :transact
-   ^{:db :asset}
-   [(merge {:asset/project project-id}
-           (asset-type-library/form->db
-            (asset-type-library/rotl-map (asset-db/asset-type-library adb))
-            asset))]})
+   (with-meta
+     (du/modify-entity-retract-nils
+      adb
+      (merge {:asset/project project-id}
```

Comment:
Do we check that the asset belongs to the project provided as argument? Do we need to?

Reply:
you mean that you could modify an existing asset and change it to the current project? Probably should check

Reply:
Yes, I assumed this can be used to also modify an existing cost item.